src/tree/ContainerAppItem.ts

```diff
@@ -82,7 +82,7 @@ export class ContainerAppItem implements ContainerAppsItem {
             id: this.id,
             label: nonNullProp(this.containerApp, 'name'),
             iconPath: treeUtils.getIconPath('azure-containerapps'),
-            contextValue: `containerApp|revisionmode:${this.containerApp.revisionsMode}`,
```

Comment:
An old artifact that could have been removed in this [PR](https://github.com/microsoft/vscode-azurecontainerapps/pull/390)